packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -43,16 +41,11 @@ import {getAggregationBits, getAggregatorModulo} from "./utils";
 export class AttestationService {
   private readonly config: IBeaconConfig;
   private readonly provider: IApiClient;
-  // order is important
-  private readonly secretKeys: SecretKey[] = [];
-  // order is important
-  private readonly publicKeys: BLSPubkey[] = [];
-  // order is important
-  private readonly validators: (ValidatorResponse | null)[] = [];
+  private readonly validators: Map<PublicKeyHex, ValidatorAndSecret> = new Map();
```

Comment:
Is there a reason that the AttestationService uses a `Map<PublicKeyHex, XXX>` but the BlockService uses a `XXX[]` to store validator pubkeys/secretkeys/etc?

Somehow I'd think we'd use one way to organize them.

Reply:
yeah that sounds like a good idea.  i'll look into it

Reply:
just pushed a fix for this